lib/Client.js

```diff
@@ -1,7 +1,8 @@
 "use strict";
-var Client = module.exports = function(options) {
-  options = options || { };
-  this._construct(options);
+var Client = module.exports = function(path, auth) {
+  path = path || { };
```

Comment:
Haha, I only touched this because Dave wanted it renamed. I'll make it throw an error.